docs/deploy/deployment-manifest.md

```diff
@@ -128,7 +128,7 @@ An example manifest for the released Industrial-IoT IoT Edge modules included in
             "status": "running",
             "restartPolicy": "always",
             "settings": {
-              "image": "mcr.microsoft.com/iotedge/opc-twin:latest",
+              "image": "mcr.microsoft.com/iotedge/discovery:latest",
               "createOptions": "{\"Hostname\":\"opctwin\",\"HostConfig\":{\"CapAdd\":[\"NET_ADMIN\"]}}"
```

Comment:
Does this also need to be changed?

Reply:
Yeah, the hostname should also change to "discovery".